group_members.go

```diff
@@ -41,6 +41,7 @@ type GroupMember struct {
 	State       string           `json:"state"`
 	CreatedAt   *time.Time       `json:"created_at"`
 	AccessLevel AccessLevelValue `json:"access_level"`
+	ExpiresAt   ISOTime          `json:"expires_at"`
```

Comment:
I think this should be `*ISOTime`... Maybe that is not needed in this case, but I prefer to keep things consistent.

Reply:
hi @svanharmelen - I changed it accordingly - sorry, my bad...